src/GovUk.Education.ExploreEducationStatistics.Admin/Models/Role.cs

```diff
@@ -0,0 +1,16 @@
+﻿#nullable enable
+using GovUk.Education.ExploreEducationStatistics.Common.Database;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Models
+{
+    // TODO EES-2462 Consider making this a class with Name and Id properties
```

Comment:
Where we want to compare by Id we get the Id by
`Role.PrereleaseUser.GetEnumValue()`

Where we want to compare by name we get the name by 
`Role.PrereleaseUser.GetEnumLabel()`

I just thought it might be nicer to be able to get them by `Role.PrereleaseUser.Id` and `Role.PrereleaseUser.Name`, but I didn't make a decision on this so left in this TODO.